jrnl/editor.py

```diff
@@ -48,15 +51,16 @@ def get_text_from_editor(config, template=""):
 
 def get_text_from_stdin():
     _how_to_quit = "Ctrl+z and then Enter" if on_windows() else "Ctrl+d"
-    print(
-        f"[Writing Entry; on a blank line, press {_how_to_quit} to finish writing]\n",
-        file=sys.stderr,
+    print_msg(
```

Comment:
![image](https://user-images.githubusercontent.com/4383304/155857979-6689f630-d817-45a8-992b-ef415925b227.png)
I think the "To finish writing" instructions should be colorized or at least formatted somehow. What do you think? See screenshot above to illustrate how input text can easily be conflated with the "To finish writing" text.